### PR TITLE
Remove amer form

### PR DESCRIPTION
**Pull request by jaesius**
## Description
Remove the AMER nerd days registrations form
Screen Shot 2020-10-15 at 9 44 49 AM

### CODE REVIEW

File: src/pages/nerd-days.js [hunk 1]
```diff
@@ -233,13 +232,16 @@ const NerdDaysPage = () => {
...
-                  Additional Nerd Days Events
-                </h3>
+              </div>
+              {/* <MarketoForm
+                id={4783}
+                title="REGISTER FOR NERD DAYS | AMERICAS"
+                munchkinId="412-MZS-894"
+                publishableKey="pk_4d10daa544de6f993a9a9ce002ccd1c6"
+                redirectLink="/nerd-days-thank-you"
+              /> */}
```
Comment (jerelmiller) [inline src/pages/nerd-days.js:242]:
```suggestion
```

Do we need any of this or would it be ok to just delete it?

**Review by jerelmiller — COMMENTED** [review]
1 minor question, otherwise this looks good!

File: src/pages/nerd-days.js (hunk 1 above)
Comment (jpvajda) [reply src/pages/nerd-days.js:242]: We can definitely delete this I think. It seems to be commented code which was removed from the form deletion

Comment (jpvajda) [reply src/pages/nerd-days.js:242]: @jaesius do you need this to stay?

Comment (jerelmiller) [reply src/pages/nerd-days.js:242]: I'm going to go ahead and commit/merge this. We can always refer back tot his PR if we need to grab some data from this code block.

**Review by jerelmiller — APPROVED** [review]

**Pull request merged**